cmd/tx.go

```diff
@@ -914,7 +925,7 @@ $ %s tx raw send ibc-0 ibc-1 100000stake cosmos1skjwj5whet0lpe65qaq4rpq03hjxlwd9
 
 			if srcChannel == nil {
 				return fmt.Errorf("could not find channel{%s} for chain{%s}@connection{%s}",
-					srcChannelID, src, path.Src.ConnectionID)
+					srcChannelID, src, path.Dst.ConnectionID)
```

Comment:
path.Dst.ConnectionID should be able to be changed to `pathConnectionID`

Reply:
I got it. I will modified it.

Reply:
@meetrick change `src` to `arg[0]`
This should solve that issue you posted. This is happening because we remapped the config and failed to update this line.

Reply:
I got it, I will modify it